aion/web/controllers/question_transactions.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Aion.QuestionTransactions do
+  @moduledoc """
+  Implements transactions used by Aion.QuestionController
+  """
+  use Aion.Web, :controller
+
+  alias Aion.Question
+  alias Aion.Answer
+  alias Aion.Subject
+  alias Ecto.Multi
+
+  @answer_separator ","
+
+  def create_question_with_answers(question, answers, subject_id) do
+      Multi.new
+      |> Multi.run(:create_changeset, fn _ -> create_question_changeset(subject_id, question) end)
```

Comment:
Well I actually tried just calling `create_question_changeset` and encountered some troubles, will try to revisit that piece of code and let you know.